src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests/Fixtures/LocationMetaGeneratorExtensions.cs

```diff
@@ -0,0 +1,56 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
+
+public static class LocationMetaGeneratorExtensions
+{
+    public static Generator<LocationMeta> DefaultLocationMeta(this DataFixture fixture)
+        => fixture.Generator<LocationMeta>().WithDefaults();
+
+    public static Generator<LocationMeta> DefaultLocationMeta(this DataFixture fixture, int options)
+        => fixture.Generator<LocationMeta>()
+            .WithDefaults()
+            .WithOptions(fixture.DefaultLocationOptionMeta().GenerateList(options));
+
+    public static Generator<LocationMeta> WithDefaults(this Generator<LocationMeta> generator)
+        => generator.ForInstance(s => s.SetDefaults());
+
+    public static Generator<LocationMeta> WithLevel(this Generator<LocationMeta> generator, GeographicLevel level)
+        => generator.ForInstance(s => s.SetLevel(level));
+
+    public static Generator<LocationMeta> WithOptions(
+        this Generator<LocationMeta> generator,
+        IEnumerable<LocationOptionMeta> options)
+        => generator.ForInstance(s => s.SetOptions(options));
+
+    public static Generator<LocationMeta> WithOptions(
+        this Generator<LocationMeta> generator,
+        Func<IEnumerable<LocationOptionMeta>> options)
+        => generator.ForInstance(s => s.SetOptions(options));
+
+    public static InstanceSetters<LocationMeta> SetDefaults(this InstanceSetters<LocationMeta> setters)
+        => setters
+            .Set(
+                m => m.Level,
+                (_, _, context) =>
+                    GeographicLevelUtils.Levels[context.Index % GeographicLevelUtils.Levels.Length]
+            )
+            .Set(m => m.Options, []);
```

Comment:
same here, do we think it would be more helpful to leave this empty?

Reply:
Similar to `FilterMeta`, we need a base `SetDefaults` that doesn't set relationships.

I've also added in the empty list as a default to `LocationMeta` itself